### DISCUSSION TITLE
Mutation testing report support

### DISCUSSION THREAD

**Discussion by rouke-broersma** [danielpalme/ReportGenerator#624] [Q&A]
We often get questions from users asking how they can merge their mutation testing reports, for example here: https://github.com/stryker-mutator/stryker-net/issues/2589

We could build this into stryker-net but our report format is shared between a couple different mutation testing tools. So we could build our own report merger tool however it's usually better to improve that which already exists (imo).

Would you be open to the mutation testing report format as a supported format? Or would that not really be compatible?
I see that ReportGenerator supports custom output formats but in this case the input format would also be custom so that doesn't work.

Our schema can be seen here: https://github.com/stryker-mutator/mutation-testing-elements/blob/master/packages/report-schema/src/mutation-testing-report-schema.json

We also provide a html report geared towards mutation testing that implements this report format but I don't think that needs to be supported as an output format for ReportGenerator to be useful to our users.

**Comment by danielpalme** [comment]
Thanks for your suggestion!
I'm not familiar with _Stryker_. But I will have a look at it as soon as possible.
Let's see if there's a good way to integrate the reports into _ReportGenerator_!
